CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 Check our main [developer changelog](https://developer.paddle.com/?utm_source=dx&utm_medium=paddle-php-sdk) for information about changes to the Paddle Billing platform, the Paddle API, and other developer tools.
 
+## [Unreleased]
+
+### Added
+
+- Added `product` to `subscription.items[]`
```

Comment:
Might be worth adding a link to [changelog](https://developer.paddle.com/changelog/2024/subscription-items-product) here 😄

Reply:
Added in https://github.com/PaddleHQ/paddle-php-sdk/pull/63/commits/9f2c32c81d56e46a26aa903cfbba9fc461c78f12